errors.go

```diff
@@ -140,6 +140,13 @@ func newFileNotFound(path string) DriverErr {
 		Parameter("path", path))
 }
 
+func logCouldNotGetOsUser(err error) {
```

Comment:
wouldn't it make sense to move it up to the other log functions, so they are grouped?

Why not have a "generic" log function?

Reply:
Good point. The problem here is that error codes are numbered and as long as we don't have an error code crawler for Go that checks the uniqueness, it's easier to have all codes in one file.

Once we have the crawler, we can inline all these functions.